### PR TITLE
Skip check for AppHost on iOS/tvOS/Android RIDs

### PR DESCRIPTION
**Pull request by akoeplinger**
They will be using a self-contained publish but without a traditional AppHost that is downloaded via a package.

### CODE REVIEW

File: src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets
Comment (dsplaisted) [inline src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets:109]: I'd suggest a different property name here.  How about `_RuntimeIdentifierUsesAppHost`?

Comment (akoeplinger) [reply src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets]: Done.

**Review by dsplaisted — APPROVED** [review]

**Comment by dsplaisted** [comment]
@akoeplinger I don't know if you have merge permissions, but if you don't feel free to ping me to merge this once CI finishes.

**Comment by akoeplinger** [comment]
@dsplaisted looks like the build passed on AzDO but the status didn't make it to GitHub: https://dev.azure.com/dnceng/public/_build/results?buildId=591912&view=results 
It'd be awesome if you could merge it :)

**Comment by akoeplinger** [comment]
It finally showed up, merged.

**Pull request merged**
